tools/upgrade_versions.py

```diff
@@ -225,7 +225,7 @@ def push_packages_head_commit(
 
 def run(CONF):
     versions_repo = setup_versions_repository(CONF)
-    packages_to_update = CONF.get('common').get('packages') or PACKAGES
+    packages_to_update = PACKAGES
```

Comment:
Not sure I follow. 

1) What is the purpose of testing with only a subset? It may work with the subset and break with a specific package, we should test with all packages.
2) Do you have a subset of packages (Git/SVN) repositories in your dev environment? I have all in mine so I can test anything I want in the build system.